src/lib.rs

```diff
@@ -1,70 +1,26 @@
 #![cfg(windows)]
 #![deny(missing_docs)]
-#![allow(clippy::drop_copy)] // we use it for discarding defer closures, makes it look nicer as a one liner
-#![no_std]
```

Comment:
any reason for removing this? The std imports you use can be done via `core`

Reply:
The no std or the drop copy? I re-added the no_std, I removed it to ease my ability to port the code without restriction, but after all is done I added it back fine.

Reply:
ye the `no_std` part (the lint is irrelevant)